shell/browser/extensions/api/resources_private/resources_private_api.cc

```diff
@@ -84,10 +46,12 @@ ExtensionFunction::ResponseAction ResourcesPrivateGetStringsFunction::Run() {
 
   api::resources_private::Component component = params->component;
 
+#if BUILDFLAG(ENABLE_PDF)
   switch (component) {
     case api::resources_private::COMPONENT_PDF:
-      AddStringsForPdf(&dict);
```

Comment:
Maybe scope `#if BUILDFLAG(ENABLE_PDF)` to just within the case `api::resources_private::COMPONENT_PDF`

```c++
case api::resources_private::COMPONENT_PDF:
#if BUILDFLAG(ENABLE_PDF)
  pdf_extension_util::AddStrings(
          pdf_extension_util::PdfViewerContext::kPdfViewer, &dict);
  pdf_extension_util::AddAdditionalData(true, false, &dict);
#endif
```